feat: add exchange management feature

s3713434 opened the pull request:
Implemented functionality to retrieve user's current exchanges and balances, convert balances into USD, and calculate the total USD value for different cryptocurrencies.

Resolve CP-14

Implemented functionality to retrieve platform names and the balance information for the user's exchanges. Balances are converted into USD, and the total USD value for different cryptocurrencies is calculated.

Delete unnecessary console log message

![NOT-FOUND](https://github.com/user-attachments/assets/8527c423-57a2-40ea-8ab1-727b67e636fb)
![SUCCESS](https://github.com/user-attachments/assets/5c4a6577-0862-4074-965b-83457d641373)

**NOTES**
Unfortunately, since I don't have any transaction history or say deposits in my Binance account, I can't give screenshots with the info :(

s3713434 committed:
feat: add exchange management feature

Implemented functionality to retrieve user's current exchanges and balances, convert balances into USD, and calculate the total USD value for different cryptocurrencies.

Resolve CP-14

Implemented functionality to retrieve platform names and the balance information for the user's exchanges. Balances are converted into USD, and the total USD value for different cryptocurrencies is calculated.

Delete unnecessary console log message

Delete unused file and adding up USD values

Added the ability to get user information directly from the context without the need for manual input.

Add exchange id into query for subsequent connection to exchange update

Add displayName into result

- Solved the problem of long fetch time
- Solved the problem of getting wrong exchangeId.

Chris-Wang reviewed (changes requested):
Need to resolve conflicts

s3713434 commented:
> Need to resolve conflicts

Just wanted to remind you that I added a little something in addition to getting the conflicts sorted out:
I added the `exchangeid` to the final object returned by `getUserExchangesAndBalances` so that it can subsequently work with the update exchange.
Please take a look and give me feedback if have any problems!

Chris-Wang reviewed (approved):
LGTM
